pkg/grpc/v1alpha1/server/server.go

```diff
@@ -22,9 +22,9 @@ import (
 	"strconv"
 
 	"github.com/golang/glog"
-	"github.com/openebs/maya/cmd/cstor-volume-grpc/api"
 	"github.com/openebs/maya/cmd/cstor-volume-mgmt/volume"
-	"github.com/openebs/maya/pkg/client/generated/cstor-volume-grpc/v1alpha1"
+	"github.com/openebs/maya/pkg/client/generated/cstor-volume-mgmt/v1alpha1"
```

Comment:
Can we have following file structure?
```
pkg/grpc/cstor-volume-mgmt/v1alpha1/server.go
pkg/grpc/cstor-volume-mgmt/v1alpha1/api.go
```

Reply:
It may be better to remove grpc as a folder altogether. e.g.
```
pkg/cstor/volume/v1alpha1/server.go
pkg/cstor/volume/v1alpha1/api.go
```

cc @kmova @sonasingh46 @prateekpandey14 What are your thoughts?

Reply:
I agree. grpc under pkg/ is confusing.